internal/cmd/configssh.go

```diff
@@ -135,6 +135,7 @@ func configSSH(configpath *string, remove *bool) func(cmd *cobra.Command, _ []st
 			fmt.Printf("Your private ssh key was written to \"%s\"\n", privateKeyFilepath)
 		}
 
+		writeSSHUXState(ctx, client)
```

Comment:
```suggestion
		_ = writeSSHUXState(ctx, client)
```

Reply:
This caused a CI error - `writeSSHUXState` as written doesn't return a value.

I initially had it return an err, but go complexity complained about too many branches.

Reply:
Oooh my bad, misread that.

Reply:
It's all good - was totally my initial intention too